File: RetroGameGauntlet.Droid/MainActivity.cs
using Android.App;
using Android.Content.PM;
using Android.OS;
using RetroGameGauntlet.PCL;
using Xamarin.Forms.Platform.Android;

namespace RetroGameGauntlet.Droid
{
    [Activity(Label = "RetroGameGauntlet.Droid",
              Icon = "@drawable/icon", 
              MainLauncher = true, 
              ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
    public class MainActivity : FormsAppCompatActivity
    {
        protected override void OnCreate(Bundle savedInstanceState)
        {
            ToolbarResource = Resource.Layout.toolbar;
            TabLayoutResource = Resource.Layout.tabs;

            base.OnCreate(savedInstanceState);

            Xamarin.Forms.Forms.Init(this, savedInstanceState);

            LoadApplication(new GauntletApp());
        }
    }
}

File: RetroGameGauntlet.Droid/MainActivity.fs
namespace RetroGameGauntlet.Droid

open System
open Android.App
open Android.Content.PM
open Android.OS
open RetroGameGauntlet.Droid
open RetroGameGauntlet.Core
open Xamarin.Forms
open Xamarin.Forms.Platform.Android

[<Activity (Label = "@string/app_name", 
            MainLauncher = true,
            Icon = "@mipmap/icon")>]
type MainActivity () =
    inherit FormsAppCompatActivity ()

    override this.OnCreate (savedInstanceState: Bundle) =
        FormsAppCompatActivity.ToolbarResource <- Resources.Layout.toolbar
        FormsAppCompatActivity.TabLayoutResource <- Resources.Layout.tabs
        base.OnCreate (savedInstanceState)
        Forms.Init(this, savedInstanceState)
        this.LoadApplication(GauntletApp())